backend/organization/serializers/status.py

```diff
@@ -34,3 +38,29 @@ def get_status_type(self, obj):
             )
         )[0][1]
         return project_status_name
+
+
+class ProjectTypesSerializer(serializers.ModelSerializer):
+    name = serializers.SerializerMethodField()
+    original_name = serializers.SerializerMethodField()
+    type_id = serializers.SerializerMethodField()
+    help_text = serializers.SerializerMethodField()
+
+    class Meta:
+        model = ProjectTypes
+        fields = ("id", "name", "original_name", "type_id", "icon", "help_text")
+
+    def get_name(self, obj):
+        return get_project_type_name(obj, get_language())
+
+    def get_original_name(self, obj):
+        return obj.name
+
+    def get_type_id(self, obj):
+        project_type_id = list(
+            filter(lambda x: x[0] == obj.type_id, ProjectTypes.POSSIBLE_PROJECT_TYPES)
+        )[0][1]
```

Comment:
Whenever there are indexes like these, it is difficult for reviewer to understand what it is doing. A better practice is to define these indexes as a variable. For example in this case the `0` index in `lambda x: x[0]` would be something like this. 
```python
        TYPE_ID_INDEX = 0
        PROJECT_TYPE_INDEX = 1
        project_type_id = list(
            filter(lambda x: x[TYPE_ID_INDEX] == obj.type_id, ProjectTypes.POSSIBLE_PROJECT_TYPES)
        )[TYPE_ID_INDEX][PROJECT_TYPE_INDEX]
```
This way it is clear for other to make sense of what is going on.